### DISCUSSION TITLE
CIDRAM Backup Ability

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#454] [General]
I like the backup ability. Very handy indeed. You even added compression. :) I think that may make it better to easily migrate over to a new version.

**Comment by Maikuolan** [comment]
> I think that may make it better to easily migrate over to a new version.

Fingers crossed. Whenever I can find the time to finally get around to writing that major version upgrade documentation (still planned, but not sure when; finding the time is always the main problem there), I plan to include using the backup feature as a major part of that process. :-)
